tb_plugin/torch_tb_profiler/profiler/event_parser.py

```diff
@@ -151,11 +154,17 @@ def _parse_node(self,
                             'Runtime and Device-op have same correlation id %s but with different external id!'
                             ' (rt external_id, device external_id): (%s, %s)' %
                             (corrid, rt_node.external_id, device_node.external_id))
-        elif event.type in [EventTypes.PYTHON, EventTypes.OPERATOR, EventTypes.PROFILER_STEP, EventTypes.MODULE]:
+        elif event.type in [EventTypes.PYTHON, EventTypes.OPERATOR, EventTypes.PL_MODULE, EventTypes.PL_PROFILE, EventTypes.PROFILER_STEP, EventTypes.MODULE]:
```

Comment:
better to move the EventTypes.PL_PROFILE to another elif section to avoid the return?

for example:

elif event.type == EventTypes.PL_PROFILE:
    pl_tid2list[int(tid)].append(PLProfileNode.create(event))

Reply:
thanks, done